IFC4x3/Properties/s/StartPointId_0OLNfz0Ir4oRaSoNkqZHEH/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="StartPointId_0OLNfz0Ir4oRaSoNkqZHEH" Name="StartPointId" UniqueId="18557a7d-012d-44c9-b91c-c97bb48d1391" PrimaryDataType="IfcIdentifier" />
```

Comment:
Is this a property? Or a relationship such as port connectivity.  It shouldn't be in a type driven override pset as I understand it.

Reply:
It's an unclear point for the Rail Telecom Domain. @czapplitec